packages/client/test/unit/util/common.test.ts

```diff
@@ -0,0 +1,42 @@
+import {isNumber} from '../../../src'
+import {expect} from 'chai'
+import {isUnsignedNumber} from '../../../src/util/common'
+
+describe('Test functions in common', () => {
```

Comment:
These tests would be good candidates for dynamic matrix tests. For example in influxdb-client-js https://github.com/influxdata/influxdb-client-js/blob/48c260c663c37e2fb6a14fb2b70925f7358bee21/packages/core/test/unit/query/flux.test.ts#L108

See also https://mochajs.org/#dynamically-generating-tests